src/components/GameSummary.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { resetGame } from '../actions';
+
+const GameSummaryPure = ({ resetGame }) => {
+  const handleClick = () => {resetGame()}
```

Comment:
Ta funcka nie jest potrzebna ;) Możesz dokładnie to samo zrobić odrazu przy onClick

Reply:
ok, poprawiam